lms/static/js/views/fields.js

```diff
@@ -143,10 +174,10 @@
                 if (xhr.status === 400) {
                     try {
                         var errors = JSON.parse(xhr.responseText),
-                            validationErrorMessage = _.escape(
+                            validationErrorMessage = HtmlUtils.ensureHtml(
```

Comment:
You don't need ensure here, because joinHtml() below will take care of escaping.  Instead, just use:
```
validationErrorMessage = errors.field_errors[this.options.valueAttribute].user_message;
```
This whole line could go away and be inlined in the other call, but I think the variable name gives a nice comment for the code without adding a comment, which can be a nice practice.